releasenotes/notes/remove-backend-v1-et-al-cccd8f6b71a97ffc.yaml

```diff
@@ -0,0 +1,73 @@
+---
+upgrade_providers:
+  - |
+    The ``BackendV1`` model has been removed following its deprecation in 
+    Qiskit 1.2.0. This includes the ``BackendV1`` class as well as related
+    modules and utils, as they have been superseded by the :class:`.BackendV2` 
+    model. The list of removed items includes:
```

Comment:
You might want to mention this also includes things that previously could accept a backend v1 object like `generate_preset_pass_manager` etc.